### ISSUE TITLE
Improve startup time

### ISSUE THREAD

**Issue by quicklizard99** [NaturalHistoryMuseum/inselect#303]
`v0.1.29` 64-bit takes

* 6.4 seconds  to start on my 2012 Mac Air and 
* 12.5 seconds to start on Windows 8.1 running virtualised on my 2012 Mac Air, 2.5 seconds for subsequent start, due to caching.

**Comment by quicklizard99** [comment]
Load large binary dependencies (`.so`, shared object, Mac OS X; `.dll` - dynamic-link library, Windows) only when required. Prime candidates are `cv2`, `numpy`, `pydmtx`, `scipy`, `sklearn`, `zbar` - these are all loaded on startup but are only needed for certain functions

**Comment by quicklizard99** [comment]
Worth noting that Windows `DLL` base addresses used to be important for startup times but have not been since Vista:

> Vista and higher versions of the Windows Operating System, ASLR (Address Space Layout Randomization) moves the image locations randomly for increased security

https://msdn.microsoft.com/en-us/library/f7f5138s.aspx

**Comment by quicklizard99** [comment]
We have had to compile some binary modules (e.g., `libzbar64-0.dll`, `libdmtx.dll`) for Windows, for which we used Visual Studio 2013 community edition. We use the [Miniconda](http://conda.pydata.org/miniconda.html) distribution, which is compiled using Visual Studio 2008.

Either rebuild the DLLs using Visual Studio 2008 or see if a Visual Studio 2013 build of Miniconda (and dependencies) is available.
